add new method to qcSummaryMongoDbservices

qinfangj opened the pull request:
Hi,have added new method for qcSummaryMongoDBServices.please confirm..

TrinidadMartin commented:
Hello Christine ;)

There is a test not working for me in SummaryMongoDBServicesSpecs: 

 "Insert 13,find 3 " should {
    "find between date1 and date2" in new TempMongoDBService {
      val ins1 = service.insert(LoadSummary("./test/resources/qc/summary1.txt").getSummaryEntry).futureValue
      ins1 must equalTo(13)

      val qcSummaryEntry = service.findAllBtw2Date("150507", "150610").futureValue
      Thread.sleep(200)
      qcSummaryEntry.size must equalTo(4)
    }

It is returning :
'10' is not equal to '4'
Expected :4
Actual   :10

Could you please check it ;)

Gooo!!! Good job!!!

Trini

qinfangj commented:
Hello,Trini,
thank you for telling the mistakes I made.
I have checked the testing source,and corrected the value of expecting data because I changed the calling method and forgot to change the testing source.
This time should go through....^^

Christine

TrinidadMartin commented:
;)
